index.html

```diff
@@ -519,6 +519,10 @@ <h2>Profiling Mechanism</h2>
         the document is using version 1.1 of the Thing Description
         specification. [[wot-thing-description11]].
       </span>
+      <span class="rfc2119-assertion" id="profiling-mechanism-5">
+        All Things and Consumers conforming to a profile MUST first satisfy the assertions
```

Comment:
```suggestion
        All Things and Consumers conforming to a profile MUST satisfy the assertions
```

Reply:
There's one requirement that needs more consideration:

TD Spec section 5.3.1.1:

> TD 1.1 consumers MUST accept TDs satisfying the W3C WoT Thing Description 1.0 [[wot-thing-description](https://www.w3.org/TR/wot-thing-description11/#bib-wot-thing-description)] specification.

Since the Profile contains additional constraints and targets to simplify the implementation, we should drop this requirement in the Profile, or change it to RECOMMENDED in the TD spec.